hy/contrib/multi.hy

```diff
@@ -21,9 +21,7 @@
 ;; DEALINGS IN THE SOFTWARE.
 
 (import [collections [defaultdict]]
-        [hy.models.expression [HyExpression]]
-        [hy.models.list [HyList]]
-        [hy.models.string [HyString]])
+        [hy [HyExpression HyList HyString]])
```

Comment:
@gilch Hy re-exports them in the main module: https://github.com/hylang/hy/blob/master/hy/__init__.py